documentation/pom.xml

```diff
@@ -59,13 +60,26 @@
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-cachestore-rocksdb</artifactId>
         </dependency>
-
         <dependency>
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-clustered-counter</artifactId>
         </dependency>
     </dependencies>
 
+    <repositories>
+      <repository>
+        <id>jcenter</id>
+        <url>https://jcenter.bintray.com/</url>
+      </repository>
+    </repositories>
+    <pluginRepositories>
+      <pluginRepository>
+        <id>jcenter</id>
+        <name>jcenter</name>
+        <url>https://jcenter.bintray.com/</url>
+      </pluginRepository>
+    </pluginRepositories>
```

Comment:
@ryanemerson and @tristantarrant the artifacts are now in maven central: https://search.maven.org/search?q=a:asciidoctorj-tabbed-code-extension 

When I build docs from maven it seems that it also downloads Infinispan artifacts from Maven Central rather than Nexus. Is that going to be OK? If not, is there a way to get around this?

Reply:
Is it because you are doing a partial build ?

Reply:
Yeah that seems to be it. 

```
mvn clean install -DskipTests
cd documentation
mvn clean package
```

Reply:
@tristantarrant Does this look OK to you now? Any other observations or nits?

Reply:
@oraNod I don't think that this `pluginRespotories` definition should be necessary. We already have `maven-settings.xml` so users can do `mvn -s maven-settings.xml clean install` etc.

Reply:
@ryanemerson right you are! I actually thought I'd removed that bug I guess not. it's gone now.